cromwell_tools/cli.py

```diff
@@ -102,4 +106,4 @@ def parser(arguments=None):
 # this should just getattr from CromwellAPI and call the func with args.
 def main(arguments=None):
     command, args = parser(arguments)
-    command(**args)
+    print(command(**args))
```

Comment:
Right now this just prints `Response<[code]>`, which isn't very useful.

Reply:
Yep I agree it's not useful to show only `Response<[code]>`. we designed the API methods to return `requests.Response` instances so that API users can have the flexibility to do anything they want with the `Response` (e.g. raise for status, retry, get raw, get json, get text, etc..) but it will be tricky for CLI users, especially when it comes to `/query`. Although the principle is to keep using the same codes for both CLI and API, I think at some point we have to write some unavoidable code for CLI specifically. But this is a good start.